pkgs/applications/misc/scli/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildPythonApplication, fetchFromGitHub, signal-cli, urwid
+, urwid-readline, dbus }:
+buildPythonApplication rec {
+  pname = "scli";
+  version = "0.6.1";
+  src = fetchFromGitHub {
+    owner = "isamert";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "10b44iky7k4bmq6bfybs53fc8mn6id1riscgpzbmzasvm320ah1g";
+  };
+  propagatedBuildInputs = [ signal-cli urwid urwid-readline dbus ];
+  dontBuild = true;
+  checkPhase = ''
+    # scli attempts to write to these directories, make sure they're writeable
+    export XDG_DATA_HOME=$(mktemp -d)
+    export XDG_CONFIG_HOME=$(mktemp -d)
+    chmod +x ./scli
```

Comment:
That +x is missing on the installed binary, right?

Reply:
nope I think that was just there when I was wrestling with it, it's removed now in 79127fe